index.html

```diff
@@ -1,38 +1,298 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <link rel="stylesheet" href="styles.css">
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;600&display=swap" rel="stylesheet">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;600&display=swap"
+      rel="stylesheet"
+    />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Portfolio</title>
-</head>
-<body>
+  </head>
+  <body>
     <header>
-        <div class="header1">
-            <h3>9:41</h3>
-            <img src="./images/mobile images.png" alt="mobile-link">
-        </div>
-        <nav class="header2">
-            <h3>Welcome</h3>
-            <img class='menu-button' src="./images/Enabled.png" alt="mobile-menu">
-        </nav>
+      <div class="header1">
+        <h3>9:41</h3>
+        <img src="./images/mobile images.png" alt="mobile-link" />
+      </div>
```

Comment:
[OPTIONAL] You actually do not need to add this div to your portfolio as it is not part of the template design but just to mimic the phone screen, your header should have a logo inclosed in `<a>` tag and a `<nav>` bar for the hamburger button only.

Reply:
Done. Thank you for your corrections